api/imapsync/read

```diff
@@ -112,7 +112,9 @@ if ($cmd eq 'list') {
     if ( glob ('/var/lib/nethserver/imapsync/*.env')) {
         $ret->{'Configured'} = JSON::true;
     }
-
+} elsif ($cmd eq 'running-info') {
+    my $out = `systemctl is-active imapsync@*`;
```

Comment:
Use `system()`, as  `$out` is not used.

Reply:
Redirection is not needed: `systemctl -q` is enough.

Reply:
ok done